workspaceitems.md

```diff
@@ -0,0 +1,113 @@
+# WorkspaceItem Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/submission/workspaceitems**   
+
+Provide access to the workspaceitems. It returns the list of existent workspaceitems.
+
+Example: to be provided
+
+## Single Item
+**/api/submission/workspaceitems/<:id>**
+
+Provide detailed information about a specific workspaceitem. The JSON response document is as follow
+```json
+{
+  "id": 1911,
+  "lastModified": "2017-06-24T00:40:54.970+0000",
+  "sections": {
+  	 "collection": "05457c63-b392-4629-a373-f2d66ee9ee33",
+  	 "traditional-page1": {
+  	 	"dc.title" : [{value: "Sample Submission Item"}],
+  	 	"dc.contributor.author" : [
+  	 		{value: "Bollini, Andrea", authority: "rp00001", confidence: 600}
+  	 	]
+  	 },
+  	 "traditional-page2": {
+  	 	"dc.subject" : [{value: "Test"}, {value: "JSON"}, {value: "REST"}],
+  	 	"dc.description.abstract" : [
+  	 		{value: "This is a long multiline text\n This is a second line of the abstract"}
+  	 	]
+  	 },
+  	 "license": {
+  	 	acceptanceDate: "2017-06-24T00:40:54.970+0000",
+  	 	url: "http://dspace7.4science.it/api/core/bitstreams/8d33bdfb-e7ba-43e6-a93a-f445b7e8a1e2/content"
```

Comment:
This looks like an excellent step towards the [generic step definitions I requested previously](https://github.com/DSpace/Rest7Contract/pull/11/commits/3fd9c24e91de383b514de0646e3f522c562e5284#r141032117), but I'm not sure the `workspaceitems` endpoint should return this information.

Reply:
yes, for the data storage in inprogress submission we agree that is better to have a generic representation.
Please check the comment https://github.com/DSpace/Rest7Contract/pull/15#discussion_r150106918 for the reason why the sections attribute is required